native/cocos/renderer/gfx-wgpu/WGPUEMSImpl.cpp

```diff
@@ -0,0 +1,164 @@
+#pragma once
+#include <emscripten/bind.h>
+#include <emscripten/val.h>
+#include <boost/fusion/include/adapt_struct.hpp>
+#include <boost/fusion/include/at_c.hpp>
+#include <boost/fusion/include/for_each.hpp>
```

Comment:
How is boost::fusion used in wgpu backend?

Reply:
Have tried kind of reflection once, deprecated. Headers of `fusion` just removed.